scripts/benchmark_k.py

```diff
@@ -288,7 +288,7 @@ def handleArgs(argv):
     'min_epochs': 25,
     'max_epochs': max_epochs,
     'auto_lr': True,
-    'max_lr': 0.0001,
+    'max_lr': 0.001,
```

Comment:
@beelze-b This is what was causing the issues with not being able to reproduce some of the results on Zeisel. I had miscopied this param from the notebook. Its always the smallest, dumbest mistake possible 🤦

Reply:
Great news is that we are more confident that we are SOTA :)